ui/src/main/webapp/app/query/operation-chain.html

```diff
@@ -13,6 +13,11 @@
     </md-button>
 </div>
 <div layout="row" layout-align="end">
-    <md-button class="md-raised overlapping" ng-click="ctrl.resetChain($event)">Reset operation chain</md-button>
-    <md-button class="md-primary md-raised overlapping" ng-disabled="!ctrl.canExecute()" id="execute-chain" ng-click="ctrl.executeChain()">Execute operation chain</md-button>
+    <md-button class="md-raised" ng-click="ctrl.resetChain($event)">Reset operation chain</md-button>
+    <md-button class="md-primary md-raised" ng-disabled="!ctrl.canExecute()" id="execute-chain" ng-click="ctrl.executeChain()">Execute operation chain</md-button>
+    <input type="text" name="operationName" ng-model="ctrl.operationName" placeholder="Operation Name"
```

Comment:
This feels out of place alongside the execute buttons etc. It may be better to bring up a sidenav. which allows users to enter their name and description before submitting the named operation. This in turn would allow us to edit the named operation in future, if we decide to introduce parameters.

Reply:
What do you mean sidenav? Do you want a whole new page on the sidebar navigation menu where you can go to save the named operation? Or do you mean bring up a popup window?

Reply:
https://material.angularjs.org/latest/demo/sidenav